app/placeholder/placeholder-deploy.sh

```diff
@@ -17,14 +17,32 @@
 # any errors? exit immediately.
 set -e
 
+# Ensure we got to the right directory. Cloud Build may start us in /workspace
+cd /app
```

Comment:
suggestion: I think it's cleaner to specify the directory in Cloud Build step than here, where setting the directory could complicate future use cases.